### ISSUE TITLE
Cant activate addon in Blender

### ISSUE THREAD

**Issue by Avatar1987** [ross-g/io_pdx_mesh#39]
![image](https://user-images.githubusercontent.com/30906293/108606373-11856c00-73ba-11eb-94f1-29ebaa43a26a.png)

Hey there,

i dont know what im doing wrong here.

**Comment by ercarp** [comment]
Mine doesn't work either. I can activate it but it doesn't appear anywhere in my Blender. There isn't even a Tool Shelf on my version of Blender (2.90.1).

**Comment by Avatar1987** [comment]
Other addons work fine

**Comment by theolaa** [comment]
There is a version that works with the latest version of blender, but it's not properly released yet. You can still use it here: https://github.com/ross-g/io_pdx_mesh/archive/bugfixes.zip

**Comment by ercarp** [comment]
> 
> 
> There is a version that works with the latest version of blender, but it's not properly released yet. You can still use it here: https://github.com/ross-g/io_pdx_mesh/archive/bugfixes.zip
> 
> You may need to open the link in new tab to get it to download. It wasn't working for me when I clicked it like normal.

Both the official release and this version give me errors. I upgraded to 2.91 of Blender as per the [author's request](https://github.com/ross-g/io_pdx_mesh/archive/bugfixes.zip#issuecomment-734514105) and now neither version of the add-on is working. I can't even activate them.

**Comment by ross-g** [comment]
The current released version doesn't work with Blender 2.91 as Blender made a change which prevents my addon from launching. This is already fixed, but not yet released... because there are other bits I'm working on that aren't ready for release yet.

@ercarp Seems like you may have misread the other issue thread #24 If you want to use the release version of this tool, then use Blender 2.90. The bugfixes version does now work with 2.91 but it currently isn't ready for download.

TLDR: downgrade to 2.9 or wait till the end of the week when I'll do another release.

**Comment by ross-g** [comment]
Duplicate of #24
